content/developer/howtos/standalone_owl_application.rst

```diff
@@ -0,0 +1,171 @@
+
+- a root component for the application
+- some setup code to start the services and mount the component
+- an assets bundle that contains the setup code
+- a QWeb view that calls the assets bundle
+- a controller that renders the view
+
+1. Root component
+=================
+
+To keep things simple, let's start with a very straightforward component that just renders
+"Hello, World!". This will let us know at a glance if our setup is working.
+
+First, create the template in `/your_module/static/src/standalone_app/root.xml`.
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <templates xml:space="preserve">
+        <t t-name="your_module.Root">
+            Hello, World!
+        </t>
+    </templates>
+
+Then create the JavaScript file for that component in `/your_module/static/src/standalone_app/root.js`.
+
+.. code-block:: js
+
+    /** @odoo-module */
+    import { Component } from "@odoo/owl";
+
+    export class Root extends Component {
+        static template = "your_module.Root";
+        static props = {};
+    }
+
+.. seealso::
+   Owl components :ref:`reference<frontend/components>`.
+
+2. Setting up the environment and mounting the application
+==========================================================
+
+An Owl application needs an environment, and the Odoo JavaScript framework needs that environment to
+contain the :ref:`services<frontend/services>`.
+Starting the services is also required to load the translations, which we need to do before mounting
+our Owl application, so that we can give Owl a working translation function.
+
+Create the JavaScript file that will mount the app in `/your_module/static/src/standalone_app/app.js`.
+
+.. code-block:: js
+
+    /** @odoo-module */
+    import { whenReady } from "@odoo/owl";
+    import { mountComponent } from "@web/env";
+    import { Root } from "./root";
+
+    whenReady(() => mountComponent(Root, document.body));
+
+
+3. Creating an assets bundle containing our code
+================================================
+
+In the manifest of your module, create a new :ref:`assets bundle<reference/assets_bundle>`.
+It should include the `web._assets_core` bundle, which contains the Odoo JavaScript
+framework and the core libraries it needs (e.g. Owl and luxon), after which you can have a
+glob that adds all the files for your application in the bundle.
+
+.. code-block:: py
+
+    {
+        # ...
+        'assets': {
+            'your_module.assets_standalone_app': [
+                ('include', 'web._assets_helpers'),
+                'web/static/src/scss/pre_variables.scss',
+                'web/static/lib/bootstrap/scss/_variables.scss',
+                ('include', 'web._assets_bootstrap'),
+                ('include', 'web._assets_core'),
+                'your_module/static/src/standalone_app/**/*',
+            ],
+        }
+    }
+
+The lines before `web._assets_core` are bundles and scss files that are required to make
```

Comment:
If that helps, you could use the [`emphasize-lines` option](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-option-code-block-emphasize-lines) of the code-block directive to simply mention "the highlighted lines" instead of "the lines before".